openprescribing/templates/measure_for_children_in_entity.html

```diff
@@ -17,6 +17,11 @@ <h1>{{measure.name}} by {{ child_entity_type_human }}s in {{ parent.name }}</h1>
     results for the individual {{ child_entity_type_human }}, plus deciles
     across all {{ child_entity_type_human }}s in the NHS in England.
   </p>
+  <p>
+    <a href="{{ parent_entity_measure_url }}" class="btn btn-default">
+      View measure as combined total for this {{ parent_entity_type_human }} &rarr;
```

Comment:
Not sure I would know what "View measure as combined total for this CCG" means.  What about "Break the measure down into individual presentations for this CCG"?  This is closer to terminology we use elsewhere.